cmd/dlv/cmds/commands.go

```diff
@@ -472,18 +472,30 @@ func dapCmd(cmd *cobra.Command, args []string) {
 	os.Exit(status)
 }
 
+func buildBinary(cmd *cobra.Command, args []string, isTest bool) string {
```

Comment:
How about making this to return (string, bool) or (string, error)? 
https://github.com/golang/go/wiki/CodeReviewComments#in-band-errors

Reply:
Good point. Done.